devices/stm32f100.yaml

```
@@ -64,6 +64,23 @@ FSMC:
     - patches/fsmc/sramfix_v3.yaml
     - fields/fsmc/sram.yaml
     - collect/fsmc/sram.yaml
+  _modify:
+    BCR1:
+      resetValue: 0x30DB
+    BCR2:
+      resetValue: 0x30D2
+    BCR3:
+      resetValue: 0x30D2
+    BCR4:
+      resetValue: 0x30D2
```

You could shorten this like:
```suggestion
    BCR[2-4]:
      resetValue: 0x30D2
```
Same below.